views/404.handlebars

```diff
@@ -1,17 +1,18 @@
 <head>
+    <base href="/">
```

Comment:
This is the key change. Setting this to `/` prevents requests like `.../quiz/1/public/assets/popsicle.svg`

Reply:
Here's a explanation of the tag: https://www.w3schools.com/tags/tag_base.asp